### PR TITLE
Implementation of Kruskal Algorithm in JavaScript

### LINKED ISSUE #2200: Kruskal Algorithm
- [ ] C - @coderprasukj
- [x] C++
- [ ] C# - @chandrikadeb7
- [ ] CoffeeScript
- [ ] Dart - @Swarnimashukla
- [ ] Go
- [x] Java
- [ ] JS - @shreyakapoor08
- [ ] Kotlin - @garimasingh128
- [ ] Php - @prince-09
- [x] Python - @Manish-cloud
- [x] Readme
- [ ] Ruby 
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by shreyakapoor08**
Fixes #2200 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
Code for Kruskal's Algorithm in Javascript

#### Languages Used:
JavaScript

#### Files Added:
Kruskal_Algorithm.js

Thanks!

### CODE REVIEW

File: Kruskal_Algorithm/Kruskal_Algorithm.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// g.addEdge("E", "G", 50);
+
+/// Sample Output
+// A->B, D
+// B->A, G
+// C->D
+// D->C, A, E
+// E->D, F
+// F->E
+// G->B
```
Comment (SakshiShreya) [inline Kruskal_Algorithm/Kruskal_Algorithm.js]: Add a new line at the EOF

File: Kruskal_Algorithm/Kruskal_Algorithm.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+       if (!uf.connected(nodes[0], nodes[1])) {
+          MST.addEdge(nodes[0], nodes[1], weight);
+          uf.union(nodes[0], nodes[1]);
+       }
+    }
+    return MST;
+ }
+
+///Sample Input
+// let g = new Graph();
```
Comment (SakshiShreya) [inline Kruskal_Algorithm/Kruskal_Algorithm.js]: Hey! I am not sure how to run this code. Can you please make it a little easier. In JS, you can write the input in the code itself.

Comment (shreyakapoor08) [reply Kruskal_Algorithm/Kruskal_Algorithm.js]: Okay I will just do that

Comment (shreyakapoor08) [reply Kruskal_Algorithm/Kruskal_Algorithm.js]: > Hey! I am not sure how to run this code. Can you please make it a little easier. In JS, you can write the input in the code itself.

I have made the changes, kindly check

**Comment by shreyakapoor08** [comment]
@SakshiShreya Kindly review

File: Kruskal_Algorithm/Kruskal_Algorithm.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
+// Javascript Implementation of Kruskals Algorithm
+
+class UnionFind {
+   constructor(elements) {
+      
+      this.count = elements.length;
+
+
```
Comment (SakshiShreya) [inline Kruskal_Algorithm/Kruskal_Algorithm.js:8]: Why have you left so many blanklines?

**Comment by shreyakapoor08** [comment]
@SakshiShreya I have made the changes. Please check

**Comment by shreyakapoor08** [comment]
@SakshiShreya Kindly review this

**Comment by shreyakapoor08** [comment]
@SakshiShreya I have squashed the commits

**Comment by shreyakapoor08** [comment]
@SakshiShreya Is it fine now ?

**Comment by shreyakapoor08** [comment]
@SakshiShreya Hii, Kindly review this PR

**Pull request merged**